src/main/java/slimeknights/mantle/registration/deferred/FluidDeferredRegister.java

```diff
@@ -79,7 +80,7 @@ public <F extends SimpleFlowableFluid> FluidObject<F> register(String name, Stri
 
     // create block and bucket, they just need a still supplier
     RegistryObject<LiquidBlock> blockObj = blockRegister.register(name + "_fluid", () -> block.apply(stillDelayed));
-    builder.bucket(itemRegister.register(name + "_bucket", () -> new BucketItem(stillDelayed.get(), ItemProperties.BUCKET_PROPS)));
+    builder.bucket(itemRegister.register(name + "_bucket", () -> new DispensibleBucketItem(stillDelayed.get(), ItemProperties.BUCKET_PROPS)));
```

Comment:
Please refrain from doing things that aren't done upstream. If you want this, make a PR for the 1.19.4 branch please

Reply:
I did that because i couldn't get the bucket interaction registration working any other way.